tests/dragonfly/seeder/script-generate.lua

```diff
@@ -33,14 +33,26 @@ LG_funcs.init(data_size, collection_size, huge_value_target, huge_value_size)
 local addfunc = LG_funcs['add_' .. string.lower(type)]
 local modfunc = LG_funcs['mod_' .. string.lower(type)]
 local huge_entries = LG_funcs["get_huge_entries"]
+local is_next_huge_entry = LG_funcs["is_huge_entry"]
+-- Keep track of total number of keys including huge value keys. Intialize
+-- to number of keys that currently exists.
+local total_keys = #keys
```

Comment:
Why not use `key_counter` ? Each time we add a key we increment it, so IMO this is the same variable with a different name.

Reply:
> monotonically increasing counter 

Uh I missed that! I thought that `total_keys` was also `decremented` on deletions!